python/ray/air/_internal/usage.py

```diff
@@ -0,0 +1,18 @@
+from typing import TYPE_CHECKING
+
+from ray._private.usage.usage_lib import TagKey, record_extra_usage_tag
+
+if TYPE_CHECKING:
+    from ray.train.trainer import BaseTrainer
```

Comment:
Maybe we should move this util to `train` package instead to avoid this reverse dependency?

In the future for generic `air` telemetry we can recreate this file here.

Reply:
no strong opinion here. Notice I put the import in function scope to not pollute the dependency tree. 

Are you suggesting a final structure that train, tune, each has its own usage file?

Reply:
the long term clean up, feels to me, is to move BaseTrainer into /ray/air/...?
like /ray/air/ defines all the interfaces, and Train has a bunch of implementations.
I remember the back & forth of where we should put these files back then. but I forgot the details.